### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by vijayee** [maidsafe/MaidSafe-Ports#1]
Could you provide build instructions for this project?

**Comment by dirvine** [comment]
Is this what you are looking for
https://github.com/maidsafe/MaidSafe/wiki

On Sun, Feb 8, 2015 at 8:10 PM, Vijayee Kulkaa notifications@github.com
wrote:

> Could you provide build instructions for this project?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/maidsafe/MaidSafe-Ports/issues/1.

## 

David Irvine
twitter: @metaquestions
blog:  http://metaquestions.me

**Comment by vijayee** [comment]
Not exactly. I am having trouble figuring out all the needed cmake variables for the ports to be created. Something similar to this command taken from the wiki:
cmake -H<path to MaidSafe root> -Bbuild_maidsafe -DCMAKE_BUILD_TYPE=Debug

**Comment by dirvine** [comment]
Ah sorry, I will let the front end guys help you out here, I am just dumb
when it comes to the important parts like this :-)

On Sun, Feb 8, 2015 at 8:26 PM, Vijayee Kulkaa notifications@github.com
wrote:

> Not exactly. I am having trouble figuring out all the needed cmake
> variables for the ports to be created. Something similar to this command
> taken from the wiki:
> cmake -H -Bbuild_maidsafe -DCMAKE_BUILD_TYPE=Debug
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/maidsafe/MaidSafe-Ports/issues/1#issuecomment-73429702
> .

## 

David Irvine
twitter: @metaquestions
blog:  http://metaquestions.me

**Comment by Viv-Rajkumar** [comment]
Hey Vijayee,

Sure, I've just added the [instructions](https://github.com/maidsafe/MaidSafe-Ports/wiki/Build-Instructions). Cheers for pointing out it's missing.

Do let us know if you hit any issues with them.

**Comment by vijayee** [comment]
Oh sweet. Thanks, I'll give this a shot later tonight.

**Comment by vijayee** [comment]
I managed to get it working and built, Thanks for the help. I additionally created a Golang port. I'm not sure what is actually being exposed in this or how to use it. What is it's relationship to the lifestuff client featured on youtube? It appears they don't share code but the lifestuff client seems to have the features those wanting to develop against the network would need.
